scripts/postbuild.js

```diff
@@ -1,125 +0,0 @@
-import assert from 'node:assert';
-import { minify } from 'terser';
-import pkg from '../package.json' assert { type: 'json' };
-import { rename, copy, write, read } from './util.js';
-
-const { version } = pkg;
-
-/**
- *
- * @param {string} file
- * @returns {Promise<string>} the compressed contents
- */
-const compress = async (file) => {
-  const minified = await minify(file, {
-    ecma: 2020,
-    module: true,
-    parse: {
-      html5_comments: false,
-    },
-    compress: {
-      drop_console: true,
-      keep_fargs: false,
-      passes: 3,
-      unsafe: true,
-    },
-    format: {
-      comments: false,
-    },
-  });
-
-  assert(typeof minified.code === 'string');
-
-  return minified.code;
-};
-
-const main = async () => {
-  /**
-   * LIBRARY POSTBUILD STEPS
-   */
-
-  // Rename our stylesheets to what the web components reference
-  rename('dist/style.css', 'dist/prime.css');
-
-  // Add Typescript definitions to the library directory
-  copy('src/prime.d.ts', 'dist/prime.d.ts');
-
-  // Add icon resources
-  copy('public/icons.woff2', 'dist/icons.woff2');
-
-  // Update icons path
-  {
-    const file = read('dist/prime.css').replaceAll(
-      /.\/icons.woff2/g,
-      'icons.woff2'
-    );
-    write('dist/prime.css', file);
```

Comment:
Unnecessary, the icons file is loaded in JS, not CSS, so this path does not appear in the CSS bundle